src/bootstrap-table.js

```diff
@@ -1670,8 +1670,8 @@
             that.trigger(e.type === 'click' ? 'click-cell' : 'dbl-click-cell', field, value, item, $td);
             that.trigger(e.type === 'click' ? 'click-row' : 'dbl-click-row', item, $tr);
 
-            // if click to select - then trigger the checkbox/radio click
-            if (e.type === 'click' && that.options.clickToSelect && column.clickToSelect) {
+            // if click to select - then trigger the checkbox/radio click unless the target is a link
+            if (e.type === 'click' && that.options.clickToSelect && column.clickToSelect && !$(e.target).is('a')) {
```

Comment:
What about other things users can click on: `<input>`, `<button>`, `<textarea>`s? I don't know how many of those stopPropagation on the click event.